src/components/organisms/AuthenticationModal/InitialForm/InitialForm.tsx

```diff
@@ -19,7 +21,7 @@ export const InitialForm: FC<InitialFormProps> = ({
       >
         Create your account
       </div>
-      <div className="buttons-separator">or</div>
+      {showSeparator && <div className="buttons-separator">or</div>}
```

Comment:
Not sure how this displays in the UI, so just want to confirm that this doesn't need any additional spacing around the "or" for it to display correctly (eg. don't want it showing up as "Create your accountorLog In")

Reply:
This is shown when you enter a venue without an account.

![login](https://user-images.githubusercontent.com/18435853/107509354-1886d000-6bab-11eb-808f-9ad8263948c7.jpg)

Reply:
Ah, cool. No issue here then :)